sass/_TextEllipsisTooltip.scss

```diff
@@ -0,0 +1,34 @@
+@import 'Variables';
+
+.coveo-text-ellipsis-tooltip {
+  font-size: $font-size-smallest;
+  background-color: $color-greyish-dark-blue;
+  color: white;
+  border-radius: 2px;
+  padding: 6px 16px;
+  position: absolute;
+  white-space: nowrap;
+
+  $arrow-width: 6px;
```

Comment:
Seems that these variables are repeated on lines 21-23? Also, I think it would be tidier to put the variables at the top of the file.